front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java

```diff
@@ -82,20 +84,27 @@ private GoogleCredential loadCredential(
   }
 
   @VisibleForTesting
-  GcsStorageService(String bucketName, String basePath, String projectName, Storage storage) {
+  GcsStorageService(String bucketName,
+                    String bucketLocation,
```

Comment:
Really this is defaultBucketLocation. It doesnt ensure the bucket is here, rather it is the default place it will create them. The potential confusion is that this has no effect on existing buckets. A comment is sufficient if you want to keep the name.